rdt/transformers/numerical.py

```diff
@@ -196,6 +196,41 @@ def _reverse_transform(self, data):
         return data.astype(self._dtype)
 
 
+class NumericalBoundedTransformer(NumericalTransformer):
```

Comment:
I think we should be adding more options here, so the different configurations can be used combining whether the data is bounded or rounded.
This one, `NumericalBounded` would possibly be equivalent to `NumericalTransformer(min_value='auto', max_value='auto')`, but keeping `rounding=None`.

Another point of discussion is the name of the Transformers. The proposed name may work, but it may also be too verbose, or confusing, once we add all the options: `NumericalBoundedTransformer`, `NumericalRoundedTransformer` and `NumericalRoundedBoundedTransfomer`.
Another option is to call them `NumericalAutoMinMaxTransformer`, `NumericalAutoRoundingTransformer` and `NumericalFullAutoTransformer`, and I'm sure there are more.

Please free to propose better names if you can come up with them @fealho

Reply:
I'll go with your bounded/rounded suggestion, it's pretty good.